### PR TITLE
UI-20 :: Fix Lack of MCap In Coin Item

### PR DESCRIPTION
**Pull request by essy16**
![Screen Shot 2023-07-08 at 12 00 08 AM]
I have an issue in the typography you can try and maybe advise me on how to.
Fixes issue #20

### CODE REVIEW

File: feature_coin_list/src/main/java/com/ryankoech/krypto/feature_coin_list/presentation/components/success/CoinItem.kt [hunk 1]
```diff
@@ -121,6 +121,8 @@ fun CoinItem(
                     color = getChangeColor(coin.change)
                 )
                 Spacer(modifier = Modifier.weight(1.0f))
+                Text(text = "MCap", style = MaterialTheme.typography.body2)
```
Comment (RyanKoech) [inline feature_coin_list/src/main/java/com/ryankoech/krypto/feature_coin_list/presentation/components/success/CoinItem.kt:125]: For this case, it more about editing the string resource for that text. Checkout the method `getFormattedMarketCap` in the component just below.

**Review by RyanKoech — COMMENTED** [review]
Thanks a lot once again. I have noted this branch For the PR has changes in the [other PR](https://github.com/RyanKoech/android-krypto/pull/71#pullrequestreview-1520887134) potentially because you branched from it branch. I'd suggest you branch from the latest develop before doing your changes

File: feature_coin_list/src/main/java/com/ryankoech/krypto/feature_coin_list/presentation/components/success/CoinItem.kt (hunk 1 above)
Comment (RyanKoech) [reply feature_coin_list/src/main/java/com/ryankoech/krypto/feature_coin_list/presentation/components/success/CoinItem.kt:125]: Oh finally, you can also checkout the contribution section of the ReadMe file to get further instruction of things such as branch naming and commit messages. Thanks!

Comment (essy16) [reply feature_coin_list/src/main/java/com/ryankoech/krypto/feature_coin_list/presentation/components/success/CoinItem.kt:125]: @RyanKoech please review and confirm

**Review by RyanKoech — CHANGES_REQUESTED** [review]
The change is simpler than it looks. To be specific, all you needed to do was add `MCap` to the string resource `coin_market_cap` changing it from `$%s %s` to `MCap $%s %s` without having to edit the code much.

Since the final else condition uses `symbol_quintillion_fallback` which is used in other places, you can go ahead to make a new resource and name it lets say `symbol_quintillion_fallback_mcap` and use it in the final else case instead since changing `symbol_quintillion_fallback` will obviously affect those other places. 

Hope that makes sense.

Finally, kindly adhere to the commit message standards as discussed earlier.

**Comment by RyanKoech** [comment]
Another thing, I have noted you have made a changes concerning PR #71 in this PR but moving the "Wallet' string to a string resource. Make sure to make keep changes within the scope of the PR or Issue you are handling at the moment.

**Comment by essy16** [comment]
@RyanKoech Please review.

**Comment by RyanKoech** [comment]
> @RyanKoech Please review.

Kindly push to this PR's branch

**Pull request closed without merging**